.github/workflows/publish-docs.yml

```diff
@@ -45,6 +45,11 @@ jobs:
           git checkout gh-pages
           cp ${{ env.ludwig_version }}/sitemap.xml latest/sitemap.xml
 
+      - name: Copy robots.txt to latest folder
+        run: |
+          git checkout gh-pages
+          cp ${{ env.ludwig_version }}/robots.txt latest/robots.xml
```

Comment:
Hey @justinxzhao , I think you have copy / paste issue here from the line above. The `robots.txt` (not `xml`) needs to be accessible at the top of the domain (aka the root directory). If you want to learn more, take a look at [Moz's Robots TXT article](https://moz.com/learn/seo/robotstxt)!
```
cp ${{ env.ludwig_version }}/robots.txt robots.txt
```

While we're at it, we could go ahead and move the `sitemap.xml` to the root directory as well. We'll need to update the robots.txt to include the updated location if we do.

Reply:
Done. Could you check the change that I just pushed for `robots.txt`?

Reply:
Perfect! LGTM!